src/View/SSViewer.php

```diff
@@ -452,20 +485,26 @@ public function dontRewriteHashlinks()
 
     /**
      * @return string
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
      */
     public function exists()
     {
+        Deprecation::notice('5.4.0', 'Will be removed without equivalent functionality to replace it');
         return $this->chosen;
     }
 
     /**
      * @param string $identifier A template name without '.ss' extension or path
      * @param string $type The template type, either "main", "Includes" or "Layout"
      * @return string Full system path to a template file
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
      */
     public static function getTemplateFileByType($identifier, $type = null)
```

Comment:
Wasn't called anywhere - and ideally people will refer to templates more loosely as candidates rather than caring about the specific files. What are you ever going to do with a template file path in code if you're not a template rendering engine?